Shakefile.hs

```diff
@@ -170,7 +176,7 @@ helpMessage = do
   out ""
   out "Targets:"
   out
-    "    build                Builds hie for all supported GHC versions (8.2.1, 8.2.2, 8.4.2, 8.4.3, 8.4.4, 8.6.1 and 8.6.2)"
+    "    build                Builds hie for all supported GHC versions (8.2.1, 8.2.2, 8.4.2, 8.4.3, 8.4.4, 8.6.1, 8.6.2 and 8.6.3)"
```

Comment:
We should eventually refactor this (and the next set of targets) to be generated from the list of `hieVersions`, so we can set that in one place only.

Reply:
in the next version of the shakefile i would like to refactor this to obtain the list of hie-versions from present stack.yaml files in the directory. But i felt like this should be in another pull request.